x-pack/plugins/security_solution/server/lib/detection_engine/signals/__mocks__/es_results.ts

```diff
@@ -910,6 +910,23 @@ export const sampleDocSearchResultsNoSortIdNoHits = (
   },
 });
 
+export const repeatedHitsWithSortId = (
```

Comment:
Probably could use a comment here explaining how to use this function. For example, what happens if the length of `destIps` doesn't match the `count` provided?